frontend/src/components/core/Sidebar/styled-components.ts

```diff
@@ -200,6 +204,12 @@ export const StyledSidebarNavLink = styled.a<StyledSidebarNavLinkProps>(
   }
 )
 
+export const StyledSidebarLinkText = styled.span<StyledSidebarNavLinkProps>(
+  ({ isActive, theme }) => ({
+    color: isActive ? theme.colors.bodyText : theme.colors.fadedText60,
+  })
+)
+
```

Comment:
would be nice to have a test that verifies that the color property changes correctly depending on the `isActive` prop

Reply:
Added on [a4a40b8](https://github.com/streamlit/streamlit/pull/5445/commits/a4a40b8a77161863dfee41de9abea53fedca2c1c)!